src/controllers/hotkeys/ActionNames.hpp

```diff
@@ -202,6 +202,13 @@ inline const std::map<HotkeyCategory, ActionDefinitionMap> actionNames{
               "[on, off, or toggle. default: toggle]",
               0,
               1,
+          }},
+         {"setShowLiveOnly",
+          ActionDefinition{
+              "Show live tabs only",
```

Comment:
Wording was a bit backwards
```suggestion
              "Only show live tabs",
```